packages/hoprd/src/api/v2/paths/channels/index.integration.spec.ts

```diff
@@ -23,9 +23,10 @@ node.openChannel = sinon.fake.returns(
 )
 
 describe('GET /channels', function () {
-  const testChannel = ChannelEntry.createMock()
-  node.getChannelsFrom = sinon.fake.returns(Promise.resolve([testChannel]))
-  node.getChannelsTo = sinon.fake.returns(Promise.resolve([testChannel]))
+  const incoming = ChannelEntry.createMock()
+  const outgoing = ChannelEntry.createMock()
```

Comment:
`ChannelEntry.createMock()` always returns the same mocked channel, so inappropriate to use here

Reply:
fixed in https://github.com/hoprnet/hoprnet/pull/4027/commits/015e05b7bb0af8a83f710d511eef7311ecc39660